toolkit/tools/internal/file/filecopybuilder.go

```diff
@@ -5,11 +5,10 @@ package file
 
 import (
```

Comment:
Chris, does this break selinux? since selinux attributes are stored in xattr?

Also, as I was working on creating an initrd from a full OS, I noticed that we need to retain things like owners and setuid/setgid/sticky bits or else booting will not complete successfully (was fun tracking what got lost on the way while copying in/out of initrd cpio).

Would it makes sense to make this behavior change parameterized? I think we want to keep as many attributes as possible unless specifically requested to ignore them...

Reply:
I don’t believe this breaks SELinux.

The existing file copy behavior only copies the permission bits and the ACLs. It doesn’t preserve other xattrs, since we don’t pass that flag. All this change does is to stop copying the ACLs.

Also, when SELinux is enabled, `setfiles` is called which will overwrite the SELinux labels for all the files in the OS. So, it doesn’t matter what the existing SELinux labels are.

For additionalFiles, we don’t really need to preserve anything other than the file contents and probably the execute bit.

For other use cases, there might be value in adding to option to preserve other properties.

Reply:
hmm... I see, I'm bit worried that when the default is to drop something, somebody will come along in the future, use the API and be surprised that something didn't copy over (I got freshly bitten by a different piece of code).
For this change, it looks like it won't affect the liveos iso dependency on `copyPartitionFiles()`.
I'll leave this to your discretion. I'll take a look at the rest of the code.